package.json

```diff
@@ -33,13 +35,14 @@
     "preprotractor": "webdriver-manager update && yarn wait",
     "protractor": "protractor  ./apps/integration-e2e/protractor.conf.js",
     "preprotractor:chrome:headless": "yarn preprotractor",
-    "protractor:chrome:headless": "yarn protractor --browser chrome --tags @desktop --headless",
+    "protractor:chrome:headless": "yarn protractor:desktop --browser chrome --headless && yarn protractor:elements --browser chrome --headless",
     "preprotractor:firefox:headless": "yarn preprotractor",
-    "protractor:firefox:headless": "yarn protractor --browser firefox --tags @desktop --headless",
-    "protractor:desktop": "yarn protractor --device=desktop --tags @desktop",
+    "protractor:firefox:headless": "yarn protractor:desktop --browser firefox --headless && yarn protractor:elements --browser firefox --headless",
+    "protractor:desktop": "yarn protractor --tags @desktop,~@elements && yarn protractor:elements",
+    "protractor:elements": "yarn protractor --feature elements --tags @elements",
```

Comment:
We probably don't need both `--feature` and `--tags` here. Should use `--tags` since tag `~@elements` is referred to in `protractor:desktop`.

Reply:
If we don't use `--feature` then saucelabs will start a vm for each feature file even if there is nothing to test in them.